templates/taobao-mini-game/game.ejs

```diff
@@ -1,4 +1,7 @@
 globalThis.__taobaoRequire = (urlNoSchema) => {
+    if (urlNoSchema.startsWith('./') && !urlNoSchema.startsWith('../')) {
```

Comment:
if `urlNoSchema.startsWith('./')` is true, then `urlNoSchema.startsWith('../')` will be false. So why should check them at the same time?

Reply:
I misunderstood the startsWith interface